ppr-ui/src/views/mhrSearch/MHRSearch.vue

```diff
@@ -4,6 +4,7 @@
       <v-progress-circular color="primary" size="50" indeterminate />
     </v-overlay>
     <v-container class="container">
+      <v-icon class="mr-2" size="30" style="vertical-align: baseline; color: #212529;">mdi-home</v-icon>
```

Comment:
Not ideal to put hex into the template, best practice to stick with the theme colour definitions. 

Sort of feel the same with regards to inline styling, why not create this into a local class. 
It's a small thing but keeping the template neat is always a good thing.

Reply:
To add to this, i know this was already done previously in the template. 
So no action required here, not expected you to clean up the other stuff, just sharing thoughts 👍

Reply:
changes made